.github/renovate.json5

```
@@ -0,0 +1,25 @@
+{
+  "extends": ["config:base", "schedule:daily", "group:allNonMajor"],
```

```suggestion
  $schema: "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:base", "schedule:daily", "group:allNonMajor"],
```

Please add the schema to it :)

---

Can you please check if vulnerabilityAlerts is activated in the base config?

---

> Can you please check if vulnerabilityAlerts is activated in the base config?

Was not sure, the docs are not that quite understandable
But I added it explicitly and configured to set the `security` label